CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
-  - UI: Fixed RBAC / Feature bugs
-  - OpenStack provider will gracefully handle 404 errors.
-  - server_monitor_poll default setting changed to 5 seconds.  This should
-    result in shorter queue wait times across the board.
-  - Fixed issue where deleting an EMS and adding it back would cause failure to
-    refresh.
-  - Fixed issue where a stopped or paused OpenStack instance could not be
+ - Server_monitor_poll default setting changed to 5 seconds, resulting in shorter queue times.
+ - Fixed issue where deleting an EMS and adding it back would cause refresh failure.
+  - EventEx is now disabled by default to help prevent event storms
+  - Fixed “High CPU usage” due to continually restarting workers when a provider is unreachable or password is invalid.
+  - RHEVM/oVirt: 
+     - Ignore user login failed events to prevent event flooding.
+     - Discovery fixed to eliminate false positives
+ - SCVMM: Fixed refresh when Virtual DVD drives are not present. 
+  - OpenStack
+       -  Image pagination issue where all of the images would not be collected.
+       -  OpenStack provider will gracefully handle 404 errors.
+       - Fixed issue where a stopped or paused OpenStack instance could not be
     restarted.
-  - More Ruby 2.0 backward compatible fixes.
-
-## Unreleased - as of Sprint 12 end 2014-09-09
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+12+Ending+Sept+9%2C+2014%22+is%3Aclosed+label%3Aenhancement)
-
-- Automate
-  - Exposed cloud relationships in automate service models.
-  - Persist state data through automate state machine retries.
-  - Moved auto-placement into an Automate state-machine step for Cloud and
-    Infrastructure provisioning.
-  - Added common "Finished" step to all Automate state machine classes.
-  - Added eligible_* and set_* methods for cloud resources to provision task
-    service model.
-- EMS Refresh / Provisioning
-  - Amazon EC2 virtualization type collected during EMS refresh for better
-    filtering of available types during provisioning.
-- UI
-  - UI updates to form buttons with Patternfly.
-- REST API
-  - Support for external authentication (httpd) against an IPA server.
-- Appliance
-  - Ability to configure a temp disk for OpenStack fleecing added to the
-    appliance console.
-  - Generation of encryption keys added to the appliance console and CLI.
-  - Generation of PostgreSQL certificates, leveraging IPA, added to the
-    appliance console CLI.
-  - Support for Certmonger/IPA to the appliance certificate authority.
+- Database
+  - Fixed seeding of VmdbDatabase timing out with millions of vmdb_metrics rows
+  - Database.yml is no longer created  after database is configured.
+  - Fixed virtual column inheritance creating duplicate entries.
+-  Appliance
+   - Fixed ftp log collection regression
+   - Ruby 2.0
+      - Fixed appliance logrotate not actually rotating the logs.
+      - Some Ruby 2.0 backward compatible fixes.
```

Comment:
This line is not needed, since it was sort of interim before we got to Ruby 2, but botvinnik is completely on 2.